pkg/helmdeployer/deployer.go

```diff
@@ -64,9 +65,10 @@ type helm struct {
 	template            bool
 	defaultNamespace    string
 	labelPrefix         string
+	labelSuffix         string
```

Comment:
hmm what's the purpose of labelSuffix?

Reply:
I see, it is for differentiating fleet-agent in local cluster so it will be using different setID to track bundle..

Reply:
Seems like it's to scope the correct SetID for the specific agent

Reply:
When the agent runs the apply code to find resources based on the setID which assumes the bundleID is globally unique for the cluster.  If I have two agents running with the same bundleID they stop on each other.  The labelSuffix is added to the setID to make them unique per agent. For upgrade we only append the labelSuffix for agents for clusters that have AgentNamespace set.